guides/micronaut-openapi-generator-server/java/.openapi-generator-ignore

```diff
@@ -0,0 +1,23 @@
+# OpenAPI Generator Ignore
```

Comment:
Should we have this file as part of the complete solution? Or should we delete it?